modules/cloud_logs/main.tf

```diff
@@ -165,15 +163,14 @@ resource "ibm_logs_router_tenant" "logs_router_tenant_instances" {
   # until provider supports passing region to this resource (coming in https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5634),
   # the for_each will only ever include the provider region
 
-  # for_each = contains(var.logs_routing_tenant_regions, "*") ? toset(data.ibm_is_regions.regions.regions[*].name) : var.logs_routing_tenant_regions
```

Comment:
oh @Aashiq-J I left that that code comment here on purpose, so we know to swithc to it when the provider support comes. You might want to add it back